src/dune/dune_file.ml

```diff
@@ -827,13 +798,11 @@ module Library = struct
          field_o "implements"
            ( Dune_lang.Syntax.since Stanza.syntax (1, 7)
            >>> located Lib_name.decode )
-       and+ variant =
-         field_o "variant"
-           ( Dune_lang.Syntax.since library_variants (0, 1)
-           >>> located Variant.decode )
+       and+ (_ : unit option) =
+         field_o "variant" (Dune_lang.Syntax.deleted_in Stanza.syntax (2, 6))
```

Comment:
this should be removed

Reply:
How about replacing this by the bare minimum to display the error message? we do that for `(jbuild_version)` for example.

Reply:
It should actually just be deleted. Thanks for pointing this out.